go-controller/pkg/libovsdbops/router.go

```diff
@@ -1005,7 +1005,10 @@ func GetRouterNATs(nbClient libovsdbclient.Client, router *nbdb.LogicalRouter) (
 	nats := []*nbdb.NAT{}
 	for _, uuid := range r.Nat {
 		nat, err := GetNAT(nbClient, &nbdb.NAT{UUID: uuid})
-		if err != nil && err != libovsdbclient.ErrNotFound {
+		if err == libovsdbclient.ErrNotFound {
```

Comment:
NIT: I would use `errors.Is(err, libovsdbclient.ErrNotFound)` in case we wrap it in the future
```
if errors.Is(err, libovsdbclient.ErrNotFound) {
```